prime-router/src/main/kotlin/history/azure/HistoryDatabaseAccess.kt

```diff
@@ -72,23 +75,38 @@ abstract class HistoryDatabaseAccess(
         klass: Class<T>,
         reportId: UUID? = null,
         fileName: String? = null,
+        receivingOrgSvcStatus: String? = null,
```

Comment:
Not a blocker, but what do you think about switching this to the enum to help catch any typo errors?